queue/init.lua

```diff
@@ -45,6 +45,8 @@ queue = setmetatable({
     end
 })
 
+queue['_VERSION'] = require('queue.version')
```

Comment:
Let's put it in the `queue` table after `cfg`.

Reply:
Done